### PR TITLE
✨ Add support for `bigquery`

### PR DESCRIPTION
**Pull request by zschumacher**
* add support of `bigquery` via `google-cloud-bigquery` client library 
* add google private key as a CI secret
* add a cursor proxy method to commands as some dbapi libs don't implement the context manager
* add two new extras
  * `google-cloud-bigquery`
  * `google-cloud-storage`
* remove custom sqlite cursor as it is now handled by the cursor proxy on commands
* remove some invalid methods on the Cursor type (not always supported)

### CODE REVIEW

File: tests/test_bigquery/conftest.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "private_key": os.getenv("GOOGLE_PRIVATE_KEY", "DUMMY").replace("\\n", "\n"),
+    "client_email": "pydapper@pydapper.iam.gserviceaccount.com",
+    "client_id": "105936813038399443987",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://oauth2.googleapis.com/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/pydapper%40pydapper.iam.gserviceaccount.com",
+}
+
+print(AUTH["private_key"])
```
Comment (matmiad) [inline tests/test_bigquery/conftest.py:27]: 🙃

**Review by matmiad — APPROVED** [review]
lgtm / works on my machine / ship it

File: tests/test_bigquery/conftest.py (hunk 1 above)
Comment (zschumacher) [reply tests/test_bigquery/conftest.py:27]: loooool

Comment (zschumacher) [reply tests/test_bigquery/conftest.py]: no chance it works on your machine lol

**Pull request merged**